packages/core/utils/compatibility.ts

```diff
@@ -1,7 +1,8 @@
 export enum GpVersion {
   V0_6_7 = "0.6.7",
   V0_7_0 = "0.7.0",
-  V0_7_1 = "0.7.1-preview",
+  V0_7_1 = "0.7.1",
```

Comment:
Versions that are not fully supported should be prefixed with `-preview`. If we fully suport a version it should be marked as the current default.